.github/actions/action-connect-ovpn/.gitignore

```diff
@@ -0,0 +1,4 @@
+.env
+*.crt
```

Comment:
should these be globbed?  `**/*key` etc?

Reply:
I would agree but eventually these will be merged into the main .gitignore so I am not sure if it's worth to abstract it or just leave it as explicit exclusions.